Deployment.md

```diff
@@ -50,5 +50,5 @@ Developer deployments are not recommended for production environments and requir
 |Developer Solutions                     |                     |                   |              |    
 | -------------------------------------- |:-------------------:|:-----------------:|:------------:|
 |Developer<br>(Testing Setup)            |Manual Install       |Manual Build       |Hardware Build|
-|**Not Recommended<br>For New Users**    |[Install From<br>Command Line](http://blog.sipsorcery.com/?p=1052)|[Build Without<br>Docker Image](https://github.com/btcpayserver/btcpayserver-doc/blob/master/ManualDeployment.md) |[ARM32v7](https://hub.docker.com/r/btcpayserver/btcpayserver/tags/)<br>[Fast Sync](https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync)<br>[BTCPayBox](https://github.com/btcpayserver/btcpayserver-doc/blob/master/HardwareDeployment.md) |
+|**Not Recommended<br>For New Users**    |[Install From<br>Command Line](http://blog.sipsorcery.com/?p=1052)|[Build Without<br>Docker Image](https://github.com/btcpayserver/btcpayserver-doc/blob/master/ManualDeployment.md) |[ARM32v7](https://hub.docker.com/r/btcpayserver/btcpayserver/tags/)<br>[Raspberry Pi](https://github.com/btcpayserver/btcpayserver-doc/blob/master/RaspberryPiDeployment.md)<br>[Fast Sync](https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync)<br>[BTCPayBox](https://github.com/btcpayserver/btcpayserver-doc/blob/master/HardwareDeployment.md) |
```

Comment:
Can you use relative path when there is a link to the page in docs repo, not absolute?
For example https://github.com/btcpayserver/btcpayserver-doc/blob/master/ManualDeployment.md should only be **ManualDeployment.md** 

This is because in GitBook, user will be redirected to GitHub and it's weird user-experience.

Of course, when it leads to another repo or external source, it can be absolute link.